content/en/blog/_posts/2024-04-02-introducing-jobset/2024-04-02-introducing-jobset.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: blog
+title: "Introducing JobSet"
+date: 2024-04-02
```

Comment:
What publication date should we target?

Reply:
Let's go for 2015-01-10 - does that work?

Reply:
yeah, that is works great.

Reply:
Set publication date for 2025-01-13 since it was a few days til I had time to address these last comments, and want to give sufficient time for a final review.